src/crypto/index.js

```diff
@@ -1126,5 +1151,13 @@ Crypto.prototype._signObject = function(obj) {
 };
 
 
+/**
+ * The parameters of a room key request. The details of the request may
+ * vary with the crypto algorithm, but the management and storage layers for
+ * outgoing requests expect it to have 'room_id' and 'session_id' properties.
```

Comment:
> the management and storage layers for outgoing requests expect it to have 'room_id' and 'session_id' properties.

Is this just a long way of saying:
```
@typedef {Object} RoomKeyRequestBody
@prop {string} room_id
@prop {string} session_id

Other keys are determined by the precise crypto algorithm used.
```
right? There may be value in `@prop`ing up the mandatory fields.